lib/cache/visitor.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Is it normal to have this sample data here?

Reply:
is it for unit tests ?

Reply:
Oh, yes not the right location. 👍